### ISSUE TITLE
FF webextension issue with injecting scriptlets

### ISSUE THREAD

**Issue by mapx-** [AdguardTeam/AdguardBrowserExtension#858]
Windows 10
FF 56 o FF nightly
english filters

- go to `olpair.com`
- click on page => popup new window

working filter in chrome + adguard, FF 55 + legacy adguard
`olpair.com#%#var adblock = function(){};`

The filter does not work (for me) in FF 56 / ninghtly + last dev of adguard (2.7.1)

**Comment by ameshkov** [comment]
Got it, thanks for reporting it!

**Comment by mapx-** [comment]
So, the working  filter in chrome + AG / FF+ AG legacy is not working anymore in AG webextension, probably due to the restrictions in webextension framework ?

**Comment by atropnikov** [comment]
This rule doesn't work in any case starting from the extension version 2.7.1. Probably, you tested this rule in chrome with the previous version of the extension.

**Comment by ameshkov** [comment]
> So, the working filter in chrome + AG / FF+ AG legacy is not working anymore in AG webextension, probably due to the restrictions in webextension framework ?

Yeah, but we've reviewed all the JS rules in AG filters prior to implementing that change.

**Comment by ameshkov** [comment]
Also, it's not just about webextensions, all the AG products now execute JS rules within an anonymous function.
